src/slic3r/GUI/Tab.cpp

```diff
@@ -3624,13 +3624,34 @@ void TabFilament::toggle_options()
     {
         bool pa = m_config->opt_bool("enable_pressure_advance", 0);
         toggle_option("pressure_advance", pa);
-        // Orca: Enable the plates that should  be visible when multi bed support is enabled or a BBL printer is selected
-        auto support_multi_bed_types = is_BBL_printer || cfg.opt_bool("support_multi_bed_types");
-        toggle_line("supertack_plate_temp_initial_layer", support_multi_bed_types );
-        toggle_line("cool_plate_temp_initial_layer", support_multi_bed_types );
-        toggle_line("textured_cool_plate_temp_initial_layer", support_multi_bed_types);
-        toggle_line("eng_plate_temp_initial_layer", support_multi_bed_types);
-        toggle_line("textured_plate_temp_initial_layer", support_multi_bed_types);
+        //Orca: Enable the plates that should  be visible when multi bed support is enabled or a BBL printer is selected
+        //auto support_multi_bed_types = is_BBL_printer || cfg.opt_bool("support_multi_bed_types");
+        // toggle_line("supertack_plate_temp_initial_layer", support_multi_bed_types );
+        // toggle_line("cool_plate_temp_initial_layer", support_multi_bed_types );
+        // toggle_line("textured_cool_plate_temp_initial_layer", support_multi_bed_types);
+        // toggle_line("eng_plate_temp_initial_layer", support_multi_bed_types);    
+        // toggle_line("textured_plate_temp_initial_layer", support_multi_bed_types);
+        DynamicConfig& proj_cfg               = m_preset_bundle->project_config;
+        std::string    bed_temp_1st_layer_key = "";
+        if (proj_cfg.has("curr_bed_type")) 
```

Comment:
Nice change!
Can you delete the commented codes, and add some comments for the new codes?